kolibri/plugins/user_profile/assets/src/machines/changeFacilityMachine.js

```diff
@@ -208,15 +218,13 @@ export const changeFacilityMachine = createMachine({
       type: 'final',
     },
     createAccount: {
+      meta: { route: 'CREATE_ACCOUNT', path: '/change_facility' },
       on: {
-        meta: { route: 'CREATE_ACCOUNT', path: '/change_facility' },
         CONTINUE: {
-          // event.value must be an object {id:, usename:}
           target: 'isAdmin',
           actions: setTargetAccount,
         },
-        BACK: 'changeFacility',
-        BACKMERGING: 'confirmMerge',
+        BACK: 'confirmAccount',
```

Comment:
I don't see this clearly looking at the figma design, could we discuss it?

Reply:
It's this step: _Confirm account username_ -> _Create new account_ so backwards it's _Create new account_ -> _Confirm account username_

![Screenshot from 2022-07-19 11-05-33](https://user-images.githubusercontent.com/13509191/179712302-590d3671-c843-4a8b-8a28-784dfa9d87f7.png)

Later on, we will need to tweak this logic to be conditional because in the _Username exists_ flow, back action will indeed need to go back to  _Change facility_, but we haven't started implementing this branch of the flow yet.

![Screenshot from 2022-07-19 11-13-25](https://user-images.githubusercontent.com/13509191/179714028-0e52ad64-0ef3-48f3-833a-b22d8fb2d299.png)

So for now, I just continued from the last state you prepared, _Confirm account username_, which is the one that belongs to _the Default path_, and made sure that this new state goes back to it. Maybe you can try to click through it on the devserver? That's a simple way to see how it works. Let me know if you need to chat about this on a call.

Reply:
@jredrejo As we discussed, I replaced `confirmMerge` by `confirmAccount`. But, I thought it'd be better rather to have only one `BACK` event and let the state machine to conditionally decide how to react to it depending on the context rather than having two back events and having `CreateAccount` page decide which one to call. Because in both of those cases, we need to check state machine's `context.accountExists` to find out where to navigate back. I think having this behavior in the state machine itself makes more sense. Please have a look at the latest update and let me know if that's looking good to you now.